operations.go

```diff
@@ -121,6 +136,21 @@ func newOpsType(stack *stackType) *opsType {
 			}
 			return []float64{float64(fact)}, 1, nil
 		}},
+
+		ophandler{"fib", "Calculate fibonacci of x", 1, func(a []float64) ([]float64, int, error) {
+			x := a[0]
+			if x-float64(uint64(x)) != 0 {
```

Comment:
With a bunch of ifs, we can use a `switch`. It's cleaner. Remember that in Go, a case is not restricted to one variable. You can do:

```
switch {
  case any_condition:
    (...)
  case any_other_condition:
    (...)
}
```

and so on.